pkg/daemon/ceph/client/filesystem.go

```diff
@@ -381,3 +381,63 @@ func GetMDSDump(context *clusterd.Context, clusterInfo *ClusterInfo) (*MDSDump,
 	}
 	return &dump, nil
 }
+
+// SubvolumeGroup is a representation of a Ceph filesystem subvolume group.
+type SubvolumeGroup struct {
+	Name string `json:"name"`
+}
+
+// SubvolumeGroupList is the representation Ceph returns when listing Ceph filesystem subvolume groups.
+type SubvolumeGroupList []SubvolumeGroup
+
+// ListSubvolumeGroups lists all subvolume groups in a given filesystem by name.
+// Times out after 5 seconds.
+var ListSubvolumeGroups = listSubvolumeGroups
+
+// with above, allow this to be overridden for unit testing
+func listSubvolumeGroups(context *clusterd.Context, clusterInfo *ClusterInfo, fsName string) (SubvolumeGroupList, error) {
+	svgs := SubvolumeGroupList{}
+
+	args := []string{"fs", "subvolumegroup", "ls", fsName}
```

Comment:
If there are hundreds or thousands of subvolume groups would this perform ok? Or if all we care about is the existence of them, perhaps there is a different command? For example, perhaps the ceph status json shows whether there are subvolume groups?

Reply:
I couldn't find any other Ceph commands that count subvolumes.

Reply:
AFAIK we dont any commands to just get the count of subvolumes